src/test/components/ng-http-loader.component.spec.ts

```diff
@@ -131,15 +131,15 @@ describe('NgHttpLoaderComponent', () => {
             const secondRequest = httpMock.expectOne('/fake2');
 
             tick();
-            component.isSpinnerVisible$.subscribe(v => expect(v).toBeTruthy());
+            component.isSpinnerVisibleSubject.subscribe(v => expect(v).toBeTruthy()).unsubscribe();
```

Comment:
You're doing the asserting directly on the subject rather than the `get isSpinnerVisible$`? Why not, but in this case you should able to use `getValue()` (or something like that if I recall the API function name correctly) rather than `subscribe`/`unsubscribe`, thanks to `BehaviorSubject`.

Reply:
Shitty refactoring, fixing now. Thanks !